notebook.ipynb

```diff
@@ -0,0 +1,1115 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "93988510-1c63-40e4-9ed9-ffa31b828e32",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Brazilian E-Commerce Public Dataset by Olist\n",
+    "\n",
+    "This is a Brazilian ecommerce public dataset of orders made at Olist Store. The dataset has information of 100k orders from 2016 to 2018 made at multiple marketplaces in Brazil. Its features allows viewing an order from multiple dimensions: from order status, price, payment and freight performance to customer location, product attributes and finally reviews written by customers.\n",
+    "\n",
+    "## Choosing a domain\n",
+    "The domain is e-commerce.\n",
+    "\n",
+    "## Describing our hypotheses and data sets\n",
+    "The dataset is composed of 6 tables (csv files): customers dataset, order items dataset, order payments dataset, orders dataset, products dataset, and product category names. They include different types of data, such as time series data, geographical data, and derived data.\n",
+    "Our hypothesis is related to customer segmentation: analyse customer demographics, such as city, to segment the customer database to identify top performing cities. This analysis can improve customer targeting."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "2cd2af13-7516-4271-bd0c-e56024e36497",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd, numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import openpyxl\n",
```

Comment:
That's not important in my opinion, just need to add to the code, that if somebody wants to run it, he/she needs to install the XlsxWriter library with the command: `pip install XlsxWriter`